integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/context/TestContext.java

```diff
@@ -76,6 +77,17 @@ public String getUuid(String uuidName) {
         return this.uuids.get(uuidName);
     }
 
+    public String getKafkaTopic(String topicName) {
+        if (!this.kafkaTopics.containsKey(topicName)) {
+            String uuidValue = UUID.randomUUID().toString().substring(0, 8);
+            String uniqueTopicName = topicName + "-" + uuidValue;
```

Comment:
A general consideration - what should be the topic name.....
Should it be identifiable as an E2E test topic? If so it should have some unique prefix.

Reply:
It the reason why the topic name starts with it-test-*.

Reply:
But you specify the prefix in the feature file in step, so it may change.
It should be ideally hardcoded.